website/static/js/pages/base-page.js

```diff
@@ -92,6 +93,100 @@ $(document).on('click', '.panel-heading', function(){
+                'If you change your mind, visit the \<a href="/settings/account/">user settings page</a>.';
+        }
+        else {
+            title = 'Add email';
+            requestMessage = 'Would you like to add \<b>' + email.address + '\</b> to your account?';
+            confirmMessage = '\<b>' + email.address + '\</b> has been added into your account.';
+            nopeMessage = 'You have chosen not to add \<b>' + email.address + '\</b> to your account. ' +
+                'If you change your mind, visit the \<a href="/settings/account/">user settings page</a>.';
+        }
+
+        var confirmFailMessage = 'There was a problem adding \<b>' + email.address +
+            '\</b>. Please contact <a href="mailto: support@osf.io">support@osf.io</a> if the problem persists.';
+
+        var cancelFailMessage = 'There was a problem removing \<b>' + email.address +
+            '\</b>. Please contact <a href="mailto: support@osf.io">support@osf.io</a> if the problem persists.';
+
+        bootbox.dialog({
+            title: title,
+            message: requestMessage,
+            onEscape: function () {
+            },
+            backdrop: true,
+            closeButton: true,
+            buttons: {
+                cancel: {
+                    label: 'Do not add email',
+                    className: 'btn-default',
+                    callback: function () {
+                        $.ajax({
+                            type: 'delete',
+                            url: confirmedEmailURL,
+                            contentType: 'application/json',
+                            dataType: 'json',
+                            data: JSON.stringify(email)
+                        }).done(function () {
+                            $osf.growl('Warning', nopeMessage, 'warning', 8000);
+                            confirmEmails(emailsToAdd.slice(1));
+                        }).fail(function (xhr, textStatus, error) {
+                            Raven.captureMessage('Could not remove email', {
+                                url: confirmedEmailURL,
+                                textStatus: textStatus,
+                                error: error
+                            });
+                            $osf.growl('Error',
+                                cancelFailMessage,
+                                'danger'
+                            );
+                        });
+                    }
+                },
+                confirm: {
+                    label: 'Add email',
+                    className: 'btn-success pull-right',
+                    callback: function () {
+                        $osf.putJSON(
+                            confirmedEmailURL,
+                            email
+                        ).done(function () {
+                            $osf.growl('Success', confirmMessage, 'success', 3000);
```

Comment:
It obscures the navbar while it's visible, requiring the user to dismiss it to reach the navbar links. But I see now that it's only visible for 3 seconds, so that mitigates a bit. 

Again, really minor, and not a blocker.